schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
 	escape_attempts int,
 	neutered bool,
 	weight_kg real,
+	species varchar(20),
```

Comment:
- [ ] Here, it is better to use a command that is going to modify the existing table for example.
`  ALTER TABLE animals ADD species varchar(20);
`
When you have an existing table with data, you would want to modify the table and not make changes that would require you to delete the data.